front-end/src/app/components/leaderboard/leaderboard.component.html

```diff
@@ -1,15 +1,12 @@
 <div class="ui inverted basic segment">
   <h1 class="ui inverted header">Leaderboard</h1>
-  <p>
-    Leaderboards may be reset when adjustments to the rating algorithm are made.
-  </p>
-
+    <p><i>Leaderboards may be reset when adjustments to the rating algorithm are made.</i></p>
   <table class="ui inverted unstackable table">
     <thead class="blue">
       <th class="center aligned">Rank</th>
       <th>Rating</th>
       <th>Username</th>
-      <th>Global PP Rank</th>
```

Comment:
It's supposed to say PP rank, to avoid confusion with website rank.

Reply:
resolved over discord